### DISCUSSION TITLE
Extracting confidence score or probability scores for extracted entities

### DISCUSSION THREAD

**Discussion by srivatsan314** [eyurtsev/kor#267] [Q&A] [answered]
Thanks for this wonderful tool. Right now I am using llama2 and I am able to extract specific fields from my text by using the correct prompts. Is there a way to give a confidence score or a probability score for each extracted entity? Right now my output looks like:

`{'data': {}, 'raw': '{\n"patient_name": "John Brown",\n"patient_dob": "09/10/1944",\n"patient_admission_number": "1112345",\n"adm_date": "11/12/1999"\n}', 'errors': [], 'validated_data': {}}`

But here I know that the `patient_admission_number` is not correct (original number is 12345). Does anyone know how to get this?

**Comment by eyurtsev** [comment, marked as answer]
Ask the model itself to provide a confidence score as part of the schema. The better the model, the better it'll work!
